doc/package.nix

```
@@ -32,16 +32,24 @@ let
     sed -i -e '/\*Declared by:\*/,+1d' $out
   '';
 in
-pkgs.stdenv.mkDerivation {
-  name = "wire-docs";
-  buildInputs = with pkgs; [
-    mdbook
-    mdbook-alerts
+pkgs.stdenv.mkDerivation (finalAttrs: {
+  pname = "wire-docs";
+  version = "3";
+  nativeBuildInputs = with pkgs; [
+    wire
+    nodejs
+    pnpm.configHook
   ];
   src = ./.;
-  buildPhase = ''
-    cat ${optionsDoc} >> ./src/modules/README.md
-    ${lib.getExe wire} inspect --markdown-help > ./src/cli/README.md
-    mdbook build -d $out
+  pnpmDeps = pkgs.pnpm.fetchDeps {
```

```suggestion
  pnpmDeps = pnpm.fetchDeps {
```

after you add `pnpm` to the top level args attrset